superduperdb/ext/unstructured/encoder.py

```diff
@@ -0,0 +1,66 @@
+import dataclasses as dc
+import pickle
+import typing as t
+
+from unstructured.documents.elements import Element
+from unstructured.partition.auto import partition
+from unstructured.partition.html import partition_html
+
+from superduperdb.components.encoder import Encoder
+
+
+def link2element(link, unstructure_kwargs):
```

Comment:
@jieguangzhou Since this is important functionality 
Can we document it (doc strings), with some references to the library.

Reply:
Done